src/file-system-cache-blob-storage.js

```diff
@@ -0,0 +1,93 @@
+var fs = require('fs-plus')
+var path = require('path')
+
+module.exports = (function () {
+  FileSystemCacheBlobStorage.load = function (directory) {
```

Comment:
What do you think about naming this class something generic like `BlobStore`, since its interface isn't specific to a caching use case?